docs/pages/management/diagnostics/tracing.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Distributed Tracing
+description: How to enable tracing within Teleport.
+---
+
+Teleport leverages [OpenTelemetry](https://opentelemetry.io/) to generate traces
+and export them to any [OpenTelemetry Protocol (OTLP)](https://opentelemetry.io/docs/reference/specification/protocol/otlp/)
+capable exporter. In the event that your telemetry backend doesn't have support receiving OTLP traces you may be able to
+leverage the [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) to proxy traces from OTLP
+to a format that your telemetry backend accepts.
+
+## Configure Teleport
+
+In order to enable tracing in Teleport, add the following section in `teleport.yaml`. For a detailed description of
+these configuration fields, see the [configuration reference](./reference/configuration.mdx) page.
+
+<Details title="Tracing Service Configuration" min="8.3.18,9.3.15,10.1" scopeOnly={true} scope={["oss", "enterprise"]}>
```

Comment:
Readers with the `cloud` will see the first paragraph of this section, then the Admonition. 

Is this only something you can set on the Auth/Proxy service? In that case, we should explain that in a Prerequisites section.

We should also remove this code fence from the `Details` box, since it should be equally visible for all readers of the guide, and I'm not sure there's a reason to collapse it.